Mage/src/main/java/mage/cards/MeldCard.java

```diff
@@ -25,6 +25,7 @@ public abstract class MeldCard extends CardImpl {
     public MeldCard(UUID ownerId, CardSetInfo setInfo, CardType[] cardTypes, String costs) {
         super(ownerId, setInfo, cardTypes, costs);
         halves = new CardsImpl();
+        this.nightCard = true;
```

Comment:
Is it a good idea to re-use nightCard for meld cards?

What does the bool indicate for actual day-night cards?